api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -209,7 +225,7 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterCreateInfraPrereq() (allErr
 	if r.Spec.ResourceGroup == nil {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("spec.resourceGroup"), r.Spec.ResourceGroup, "value of resource group is empty"))
 	}
-	if err := r.validateIBMPowerVSClusterVPCSubnetNames(); err != nil {
```

Comment:
We also need to validated duplicate subnet names right?

Reply:
I have added it here already- https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/76b57c8aeb9256bbb66b3a5a01779ec4c9934a71/api/v1beta2/ibmpowervscluster_webhook.go#L156-L158